modules/fbs-core/web/src/app/page-components/sql-playground/sql-input-tabs/sql-input-tabs.component.html

```diff
@@ -0,0 +1,28 @@
+<app-bordered-container>
```

Comment:
<app-bordered-container> should be implemented in "sql-playground.component"

The component does not belong to a spezific input tab and should be configured in the main component.

```
<app-bordered-container>
     <app-sql-input-tabs></app-sql-input-tabs>
</app-bordered-container>
```